app/src/app/components/contents/PacketGroup/PacketGroup.tsx

```diff
@@ -1,19 +1,48 @@
-import { useParams } from "react-router-dom";
 import { PacketTable } from "./PacketTable";
+import {useParams} from "react-router-dom";
```

Comment:
formatting issues? there is no space b/w word and brackets

Reply:
Which formatter is meant to catch this?

Reply:
I'd have thought prettier would have picked that up.

Reply:
Cool, I enabled prettier in my code editor and it picked this up.